conf/nginx.conf

```diff
@@ -4,88 +4,179 @@ set $allowed_origins "https://${sandbox_domain}";
 set $api_domain "__DOMAIN__";
 set $files_domain "__DOMAIN__";
 ssl_ecdh_curve secp384r1;
-more_set_headers "Access-Control-Allow-Origin: '${allowed_origins}'";
+
+more_set_headers "X-XSS-Protection: 1; mode=block";
+more_set_headers "X-Content-Type-Options: nosniff";
```

Comment:
```suggestion
```

These are already part of Yunohost's conf : https://github.com/YunoHost/yunohost/blob/dev/conf/nginx/security.conf.inc#L33